packages/pigeon/lib/pigeon_lib.dart

```diff
@@ -118,37 +122,40 @@ bool _isFlutterApi(ClassMirror apiMirror) {
 /// Options used when running the code generator.
 class PigeonOptions {
   /// Path to the file which will be processed.
-  String input;
+  String? input;
```

Comment:
We could make this non-null if we wanted.  The command line parser enforces that it is present.  Feel free to leave it as is and I can update it some other time.

Reply:
I reverted this change, sorry.  It had unintended consequences.  When you run pigeon without arguments, it's fine for this to bo null.